src/adapter/etl-adapter-google-sheet/src/Flow/ETL/Adapter/GoogleSheet/Columns.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Adapter\GoogleSheet;
+
+use Webmozart\Assert\Assert;
+
+final class Columns
+{
+    public function __construct(
+        public readonly string $sheetName,
+        public readonly string $startColumn,
+        public readonly string $endColumn,
+    ) {
+        Assert::notEmpty($sheetName);
+        Assert::unicodeLetters($startColumn);
```

Comment:
could you replace Assert with regular if statements? This assertion library is not a direct project dependency, it's available through some other dependencies but I would like to keep core dependencies as few as possible